public/index.html

```diff
@@ -39,5 +39,7 @@
       To begin the development, run `npm start` or `yarn start`.
       To create a production bundle, use `npm run build` or `yarn build`.
     -->
+    <!-- This is to import the SDK -->
```

Comment:
We can remove this comment now

Reply:
Just deleted this Comment, and have started working on tests.